packages/teams-js/test/private/privateAPIs.spec.ts

```diff
@@ -581,6 +581,7 @@ describe('AppSDK-privateAPIs', () => {
       viewerAction: ViewerActionTypes.view,
       fileOpenPreference: FileOpenPreference.Web,
       conversationId: 'someConversationId',
+      size: 'someSize',
```

Comment:
since it is optional, not sure if we want to have two test cases, one is with this param(as being done here) and the other one is without this parameter (as it was before)

Reply:
It's a good idea to be comprehensive but in this specific case I'm not sure that adding a test that _doesn't_ have the property is adding any new value, as it seems like practically all the properties on `FilePreviewParameters` are optional, so it seems like testing that ANY optional parameter is handled correctly would be the same as testing any other optional parameter

That said it seems like there aren't any tests that validate the right thing happens for any optional parameter, so maybe this is the time to add a test for that (although I would probably try it with one of the optional parameters in the middle of the list rather than at the end like `size`)

Reply:
Added size in the middle of the list. @AE-MS is this ok or do you want me to add more tests ?

Reply:
This is okay for this change!